doc/rtd/explanation/kernel-cmdline.rst

```diff
@@ -3,20 +3,48 @@
 Kernel command line
 *******************
 
+Providing configuration data via the kernel command line is somewhat of a last
+resort, since this method supports only a subset of the
```

Comment:
"subset of the cloud config format" reads a bit weirdly. What does it mean?

Reply:
It means that this doesn't actually support all cloud config user data formats. Some valid user data cloud configurations may start with `Content-Type: text/x-shellscript` in the case of multipart MIME archive, however this isn't supported at this time via this interface - therefore only a subset of the cloud config format is supported.